pkgs/by-name/bl/blanket/package.nix

```diff
@@ -35,6 +36,7 @@ python3Packages.buildPythonApplication rec {
   ];
 
   buildInputs = [
+    blueprint-compiler
```

Comment:
Should be fine in the nativeBuildInputs, its just a compiler not a runtime dependency.

Reply:
Sure thing 🙂 Apologies if I missed that from your previous comment.